pkgs/by-name/ca/cargo-embassy/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+let
+  version = "0.3.4";
+in
+rustPlatform.buildRustPackage {
+  pname = "cargo-embassy";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "adinack";
+    repo = "cargo-embassy";
+    tag = "${version}";
+    hash = "sha256-/v4qgoXHkhJyD22GG0sRR32CcEYNOhmCSG5qDS7Csk4=";
+  };
+
+  # Upstream doesn't commit Cargo.lock
```

Comment:
Please ask upstream to include Cargo.lock. We try to avoid storing lockfiles in nixpkgs.

Reply:
Did this here: https://github.com/AdinAck/cargo-embassy/issues/31